events.go

```diff
@@ -133,6 +133,9 @@ func (s *Session) handleNodeEvent(frames []frame) {
 		// TODO: can we be sure the order of events in the buffer is correct?
 		switch f := frame.(type) {
 		case *topologyChangeEventFrame:
+			host, port := s.cfg.translateAddress(f.host.String(), f.port)
```

Comment:
That's probably fair. In general we were trying to avoid altering too many of the existing structures and behaviors. Especially for testing reasons. But, keeping both addresses (broadcast and translated) makes sense.